File: src/main/kotlin/agendaonline/model/Usuario_.java
package agendaonline.model;

import javax.persistence.metamodel.SingularAttribute;

public class Usuario_ {
    public static volatile SingularAttribute<Usuario, Long> id;
    public static volatile SingularAttribute<Usuario, String> nome;
    public static volatile SingularAttribute<Usuario, String> email;
    public static volatile SingularAttribute<Usuario, String> senha;
    public static volatile SingularAttribute<Usuario, String> celular;
    public static volatile SingularAttribute<Usuario, String> foto;
    public static volatile SingularAttribute<Usuario, Boolean> ativo;
    public static volatile SingularAttribute<Usuario, Cliente> cliente;
}

File: src/main/kotlin/agendaonline/model/Usuario.kt
package agendaonline.model

import com.fasterxml.jackson.annotation.JsonIgnore
import com.fasterxml.jackson.annotation.JsonRootName
import org.hibernate.validator.constraints.NotBlank
import org.jetbrains.annotations.NotNull
import javax.persistence.*

@Entity
@JsonRootName("usuario")
data class Usuario(
        @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
        var id:Long = 0L,
        @NotNull var nome:String = "",
        @NotNull var email:String = "",
        @JsonIgnore
        @NotNull @NotBlank var senha: String = "",
        @NotNull var celular:String = "",
        var foto:String = "",
        @JsonIgnore
        @NotNull var ativo:Boolean = false,
        @JsonIgnore
        @OneToMany(fetch = FetchType.LAZY, mappedBy = "usuario")
        var cliente: MutableList<Cliente> =  mutableListOf()
){
    override fun toString(): String = "Usuario_($nome,$email)"
}